f5-distributed-cloud/README.md

```diff
@@ -0,0 +1,99 @@
+## Overview
+
+F5 Distributed Cloud (XC) Services provides a global cloud native platform where customers can deploy, manage, and secure their applications regardless of whether the application resides in a public cloud, in a private data center, or a colocation facility (see below). It provides a variety of both ADN and CDN services.
+
+![snapshot][1]
+
+Although the F5 XC console UI provides very good observability natively, many enterprises prefer to aggregate their telemetry from various sources and centralize visibility/analytics on the Datadog platform.  To this end, the F5 XC platform includes the Global Log Receiver.
+
+A F5 XC global log receiver can be configured to securely send logs to a Datadog HTTPS logging endpoint. Configuration is quite simple and can be done via the [F5 XC Console UI][6].
+
+In addition to the following configuration guidance. The F5 Distributed Cloud Services integration includes:
+
+- Dashboard - *Access Log Overview*
+- Saved View - *Including facets for commonly queried fields*
+
+![snapshot][2]
+
+## Setup
```

Comment:
Your setup section includes a lot of information/wordy. We recommend cutting down on the detail and consolidate steps if possible